tests/conftest.py

```diff
@@ -54,93 +54,153 @@ def __init__(self):
+                        0.04055141
                     ],
                     "thrust": [
-                        1.10610965,
-                        1.09515807,
-                        1.0227122,
-                        0.9196487,
-                        0.85190470,
-                        0.80328229,
-                        0.76675469,
-                        0.76209299,
-                        0.76209299,
-                        0.75083241,
-                        0.67210674,
-                        0.52188504,
-                        0.43178758,
-                        0.36443258,
-                        0.31049874,
-                        0.26696686,
-                        0.22986909,
-                        0.19961578,
-                        0.17286245,
-                        0.15081457,
-                        0.13146666,
-                        0.11475968,
-                        0.10129584,
-                        0.0880188,
-                        0.07746819,
-                        0.06878621,
-                        0.05977061,
-                        0.0,
+                        1.19187945,
+                        1.17284634,
+                        1.09860817,
+                        1.02889592,
+                        0.97373036,
+                        0.92826162,
+                        0.89210543,
+                        0.86100905,
+                        0.835423,
+                        0.81237673,
+                        0.79225789,
+                        0.77584769,
+                        0.7629228,
+                        0.76156073,
+                        0.76261984,
+                        0.76169723,
+                        0.75232027,
+                        0.74026851,
+                        0.72987175,
+                        0.70701647,
+                        0.54054532,
+                        0.45509459,
+                        0.39343381,
+                        0.34250785,
+                        0.30487242,
+                        0.27164979,
+                        0.24361964,
+                        0.21973831,
+                        0.19918151,
+                        0.18131868,
+                        0.16537679,
+                        0.15103727,
+                        0.13998636,
+                        0.1289037,
+                        0.11970413,
+                        0.11087113,
+                        0.10339901,
+                        0.09617888,
+                        0.09009926,
+                        0.08395078,
+                        0.0791188,
+                        0.07448356,
+                        0.07050731,
+                        0.06684119,
+                        0.06345518,
+                        0.06032267,
+                        0.05741999,
+                        0.05472609
                     ],
                     "wind_speed": [
-                        0.0,
```

Comment:
Is there a reason to change this from 0.0 to 2.0? I feel like 0.0 covers more cases that people may try, or at least would cover more cases for the interpolation.